docs/oauth2-oidc/wellknown-endpoint-discovery.mdx

```diff
@@ -0,0 +1,834 @@
+---
+id: wellknown-endpoint-discovery
+title: OAuth2.0 & OpenID Connect well-known endpoint discovery
+sidebar_label: OAuth2.0 & OpenID Connect well-known endpoint discovery
+---
+
+There are two types of well-known discovery endpoints. The OAuth2.0 and the OpenID Connect (ODIC) discovery endpoints. Both
+endpoints return nearly identical information as OIDC is built on top of OAuth 2.0 (OIDC is a superset of OAuth 2.0).
```

Comment:
```suggestion
endpoints return nearly identical information.
```

imo we dont need to explain OIDC/OAuth2 here, I would assume the reader already know this - alternatively link to the document that explains this in detail?